### ISSUE TITLE
Doesn't work on iOS platform

### ISSUE THREAD

**Issue by riodiep** [hansemannn/titanium-crashlytics#50]
I followed the instructions and only worked on Android but not iOS. It didn't show up in Crashlytics Firebase console. Analytics is showing up.

Below are details of my setup:

```
Ti SDK:               10.2.0.v20211006024318
Alloy:                1.11.0
iOS:                  15.0 Simulator
Xcode:                13.0
Firebasecore:         7.2.0 
FirebaseAnalytics:    6.2.0
FirebaseCrashlytics:  4.3.0
```

File locations:
- rootApp\app\assets\iphone\GoogleService-Info.plist
- rootApp\scripts\run
- rootApp\scripts\upload-symbols

alloy.js
```
var FirebaseCore = require('firebase.core');
var FirebaseAnalytics = require ('firebase.analytics');
 
FirebaseCore.configure();

var Alloy.Globals.Crashlytics = require('ti.crashlytics');
```

Tab.js
```
$.tab.addEventListener('open', function() {
         var a = undefined;
         var t = a[1];
 });
```

**Comment by williamycxue** [comment]
@hansemannn could you please confirm that Crashlytics does not work in Simulators right? The hooks are no injected for non production build.

**Comment by hansemannn** [comment]
This may have been fixed by #48, a new release is ready today!
